api/mesh/v1alpha1/dataplane.proto

```diff
@@ -163,7 +174,7 @@ message Dataplane {
     Gateway gateway = 3;
 
     // Inbound describes a list of inbound interfaces of the dataplane.
-    repeated Inbound inbound = 1;
+    repeated Inbound inbound = 1 [ (doc.required) = true ];
```

Comment:
I think in the case of a gateway there's no inbound

Reply:
addressed